src/labels/CustomLabels.labels

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomLabels xmlns="http://soap.sforce.com/2006/04/metadata">
+    <labels>
+        <fullName>TelemetryParamException</fullName>
+        <categories>Exceptions</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Unsupported metric value.</shortDescription>
```

Comment:
@katherinemitchell - This error should hopefully never occur but is there as a safety precaution and would only show up under Setup -> Apex Jobs -> If the queueable fails.

Reply:
This would be if somehow we attempted to send a feature param that was not an integer like a boolean or date or other unsupported data types. We are currently only using Integers.

Reply:
line 9 - "The data type being submitted is not currently being collected."